### DISCUSSION TITLE
Error: cannot find module 'pify'

### DISCUSSION THREAD

**Discussion by nbilyk** [gulpjs/gulp#2533] [Help]
This may be a fluke npm error, but reporting it here in case others hit the same problem.

When I run gulp through node, I get the error:

Error: cannot find module 'pify'

(in package.json I have in scripts: "dist": "gulp")

```
$ npm run-script dist

> gyre@0.1.0 dist /Users/nbilyk/Documents/projects/gyre
> gulp

internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module 'pify'
Require stack:
- /Users/nbilyk/Documents/projects/gyre/node_modules/load-json-file/index.js
- /Users/nbilyk/Documents/projects/gyre/node_modules/read-pkg/index.js
- /Users/nbilyk/Documents/projects/gyre/node_modules/read-pkg-up/index.js
- /Users/nbilyk/Documents/projects/gyre/node_modules/yargs/yargs.js
- /Users/nbilyk/Documents/projects/gyre/node_modules/yargs/index.js
- /Users/nbilyk/Documents/projects/gyre/node_modules/gulp/node_modules/gulp-cli/index.js
- /Users/nbilyk/Documents/projects/gyre/node_modules/gulp/bin/gulp.js
```

I don't get this error through gulp-cli, only npm.
I can resolve this problem by installing ```pify npm install --save-dev pify```
pify existed in node_modules after installing gulp, but was empty with just a package.json.
MacOS Catalina 10.15.3
Node v14.15.3
NPM 6.14.9
Gulp CLI: 2.3.0

**Comment by JhonCodeU** [comment]
thank you, its work
